PWGMM/Mult/Tasks/heavy-ion-mult.cxx

```diff
@@ -0,0 +1,214 @@
+// 1. https://github.com/AliceO2Group/O2Physics/blob/master/PWGMM/Mult/Tasks/dndeta.cxx
+// 2. https://github.com/AliceO2Group/O2Physics/blob/master/PWGMM/Mult/Tasks/dndeta-hi.cxx
+// 3. https://github.com/AliceO2Group/O2Physics/blob/master/PWGLF/Mult/Tasks/puremc-dndeta.cxx
+// 4. O2 analysis tutorial: https://indico.cern.ch/event/1267433/
+
+#include <cmath>
+#include <cstdlib>
+#include <iostream>
+#include <TPDGCode.h>
+#include <TDatabasePDG.h>
+
+#include "bestCollisionTable.h"
+#include "CCDB/BasicCCDBManager.h"
+#include "Common/Core/trackUtilities.h"
+#include "Common/CCDB/EventSelectionParams.h"
+#include "Common/Core/TrackSelection.h"
+#include "Common/DataModel/Centrality.h"
+#include "Common/DataModel/EventSelection.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "CommonConstants/MathConstants.h"
+#include "Framework/ASoAHelpers.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/Configurable.h"
+#include "Framework/O2DatabasePDGPlugin.h"
+#include "Framework/runDataProcessing.h"
+#include "ReconstructionDataFormats/GlobalTrackID.h"
+#include "ReconstructionDataFormats/Track.h"
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+using namespace o2::aod::track;
+using CollisionDataTable = soa::Join<aod::Collisions, aod::EvSels>;
+using TrackDataTable = soa::Join<aod::Tracks, aod::TracksExtra, aod::TracksDCA, aod::TrackSelection>;
+using FilTrackDataTable = soa::Filtered<TrackDataTable>;
+using CollisionMCTrueTable = aod::McCollisions;
+using TrackMCTrueTable = aod::McParticles;
+using CollisionMCRecTable = soa::SmallGroups<soa::Join<aod::McCollisionLabels, aod::Collisions, aod::EvSels>>;
+using TrackMCRecTable = soa::Join<aod::Tracks, aod::TracksExtra, aod::TracksDCA, aod::McTrackLabels, aod::TrackSelection>;
+using FilTrackMCRecTable = soa::Filtered<TrackMCRecTable>;
+static constexpr TrackSelectionFlags::flagtype trackSelectionITS =
+  TrackSelectionFlags::kITSNCls | TrackSelectionFlags::kITSChi2NDF |
+  TrackSelectionFlags::kITSHits;
+static constexpr TrackSelectionFlags::flagtype trackSelectionTPC =
+  TrackSelectionFlags::kTPCNCls |
+  TrackSelectionFlags::kTPCCrossedRowsOverNCls |
+  TrackSelectionFlags::kTPCChi2NDF;
+static constexpr TrackSelectionFlags::flagtype trackSelectionDCA =
+  TrackSelectionFlags::kDCAz | TrackSelectionFlags::kDCAxy;
+AxisSpec axisEvent{4, -0.5, 3.5, "#Event"};
+AxisSpec axisVtxZ{800, -20, 20, "Vertex Z"};
+AxisSpec axisDCA = {601, -3.01, 3.01};
+AxisSpec axisPT = {1000, -0.05, 49.95};
+AxisSpec axisMult{2000, -0.5, 1999.5, "Multiplicity"};
+AxisSpec axisEta{200, -5, 5, "#eta"};
+AxisSpec axisMCEvent_ambiguity{6, -0.5, 5.5, "reco collisions per true collision"};
+struct HeavyIonMultiplicity {
+  HistogramRegistry histos{"histos", {}, OutputObjHandlingPolicy::AnalysisObject};
+  Service<O2DatabasePDG> pdg;
```

Comment:
Please use `o2::framework::O2DatabasePDG` as there is an ambiguity here

Reply:
Hi Anton, Thanks a lot for pointing out this.
Now it is done.